docs/memory.md

```diff
@@ -79,6 +86,47 @@ ThisFeature.test_adapter.on!(:flag_name, context: user) # with context
 ThisFeature.test_adapter.off!(:flag_name)               # without context
 ```
 
+### **#enable_treatment!**
+
+This method is useful when you need to enable a feature flag with a treatment (or multiple treatments), and not just `"on"` and `"off"`.
+
+Usage example of these:
+
+```ruby
+# If you have configured the in-memory adapter as the default
+ThisFeature.test_adapter.enable_treatment!(:flag_name, treatment: 'treatment_name', context: user)
```

Comment:
Not for this PR, but I did notice it matter whether or not you pass the flag name as a symbol or string, in that it must be retrieved from the storage in the same way. May be a good for `.with_indifferent_access` on the storage hash